src/Text/LLVM/AST.hs

```diff
@@ -1271,22 +1271,31 @@ data DICompositeType' lab = DICompositeType
   , dictAssociated     :: Maybe (ValMd' lab)
   , dictAllocated      :: Maybe (ValMd' lab)
   , dictRank           :: Maybe (ValMd' lab)
+  , dictAnnotations    :: Maybe (ValMd' lab)
+    -- ^ Introduced in LLVM 14.
   } deriving (Data, Eq, Functor, Generic, Generic1, Ord, Show, Typeable)
 
 type DICompositeType = DICompositeType' BlockLabel
 
 data DIDerivedType' lab = DIDerivedType
-  { didtTag :: DwarfTag
-  , didtName :: Maybe String
-  , didtFile :: Maybe (ValMd' lab)
-  , didtLine :: Word32
-  , didtScope :: Maybe (ValMd' lab)
-  , didtBaseType :: Maybe (ValMd' lab)
-  , didtSize :: Word64
-  , didtAlign :: Word64
-  , didtOffset :: Word64
-  , didtFlags :: DIFlags
-  , didtExtraData :: Maybe (ValMd' lab)
```

Comment:
Generally, I prefer non-aligned type ascriptions for the reason that, as in this PR, adding a longer field name causes a diff that includes *all* of the fields, making it harder to distinguish what's changed. That being said, it looks like this changed to be more consistent with other parts of the file, which is also important.

Also looks like maybe I did this? :sweat_smile: So I can't complain too much.

Reply:
Indeed, this is a cherry-picked version of https://github.com/elliottt/llvm-pretty/pull/37, so I blame you for this :)

Then again, I have already performed some surgery on that commit, so I would be fine with removing this particular part of the commit (or factoring it out into a separate commit).

Reply:
I'm fine with whatever, not a big deal either way!

Reply:
I opted to just revert the other alignment-related changes, as this file never consistently uses one convention for all its data type declarations.